libsrc/interface/nginterface.cpp

```diff
@@ -118,122 +118,122 @@ void Ng_LoadMeshFromStream ( istream & input )
 
 void Ng_LoadMesh (const char * filename)
 {
-  {
```

Comment:
why do you comment this code ?

Reply:
I think it was a problem with gcc4.8. This is more a problem with conda. So for sure this shouldn't be merged. I just wanted to start a little discussion to see which stuff has potential to be accepted...